website/source/api/secret/nomad/index.html.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: "api"
+page_title: "Nomad Secret Backend - HTTP API"
+sidebar_current: "docs-http-secret-nomad"
+description: |-
+  This is the API documentation for the Vault Nomad secret backend.
+---
+
+# Nomad Secret Backend HTTP API
+
+This is the API documentation for the Vault Nomad secret backend. For general
+information about the usage and operation of the Nomad backend, please see the
+[Vault Nomad backend documentation](/docs/secrets/nomad/index.html).
+
+This documentation assumes the Nomad backend is mounted at the `/nomad` path
+in Vault. Since it is possible to mount secret backends at any location, please
+update your API calls accordingly.
+
+## Configure Access
```

Comment:
GET for `Configure Access` and `Configure Lease` should be documented.

Reply:
Fixed in ea66973fc